doc/neps/roadmap.rst

```diff
@@ -97,6 +97,9 @@ first, and that could potentially be upstreamed into NumPy later, include:
   ``latin1``)
 - A unit dtype
 
+We further plan to extend the new API around ufuncs as the needs arise.
+One possibility here is creating a new, more powerful, API to allow hooking
+into existing NumPy ufunc implementations.
```

Comment:
Note sure anyone can understand it, but I don't mind :).  I am talking about `pnumpy` here, we should have a better story for it (or decide to not have a story at all).

Reply:
Change "new API around ufuncs" to "ufunc C API"?

Reply:
Thanks, better.

Reply:
I had never heard of `pnumpy`, but it seems interesting. Maybe add at end "... implementations, e.g., to allow parallel execution as done by [``pnumpy``](https://quansight.github.io/pnumpy/stable/index.html)"

Reply:
Not sure how maintained it still is (or quite how well it worked in practice), but if others think it is stable enough to link, sure.
(Also, parallel execution belongs on a different level IMO, but that is part of the potential things here.)

Reply:
It's not maintained. It was an experiment, it worked and yielded interesting performance gains, but it wasn't interesting enough to pursue further at that point. I think it is an example of the types of things people would like to be able to do though, so it can serve as an example use case I'd say. @mattip may have more thoughts on it.

Reply:
If it is not pursued anymore then linking to it is perhaps not such a good idea... But it helps to give some example of why linking into the implementations might be useful.

Reply:
pnumpy is unmaintained. I wouldn't want to draw more attention to it.